masterlist.yaml

```diff
@@ -1085,6 +1085,8 @@ plugins:
   - name: 'Update.esm'
     group: *dlcGroup
     tag:
+      - C.Encounter
```

Comment:
You won't need that if USSEP is being used, or if the `Dawnguard.esm` wild edits have been cleaned.